pages/02.documentation/06.node-deployments/default.en.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Node Deployments
+published: true
+taxonomy:
+    tag:
+        - nodes
+        - kubernetes
+---
+
+## Node Deployments
```

Comment:
This is partly a duplication of https://docs.syseleven.de/metakube/de/documentation/cluster-api

Reply:
Does this need to be updated as well?